meta-rcar-gen3-xen/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -1,10 +1,20 @@
 #Add Xen to build
 IMAGE_INSTALL_append = " \
     xen-base \
-    xen-flask \
```

Comment:
Adding flask unveils dilemma of the XEN_REL value. Xenpolicy file name is extended by current XEN version (from xenbits/master it is 4.10-unstable now). Thus with xenbits/master version progress the recipe should be changed to let yocto build not fail finding xenpolicy file.